pkg/executor/infoschema_reader.go

```diff
@@ -3466,8 +3466,8 @@ func (e *memtableRetriever) setDataFromRunawayWatches(sctx sessionctx.Context) e
 		row := types.MakeDatums(
 			watch.ID,
 			watch.ResourceGroupName,
-			watch.StartTime.Local().Format(time.DateTime),
-			watch.EndTime.Local().Format(time.DateTime),
+			watch.StartTime.Format(time.DateTime),
+			watch.EndTime.Format(time.DateTime),
```

Comment:
```suggestion
			watch.StartTime.UTC().Format(time.DateTime),
			watch.EndTime.UTC().Format(time.DateTime),
```
Better always ensure the value is formatted under the UTC timezone.